docs/design.md

```diff
@@ -40,9 +40,9 @@ case today. The currently *suggested* purpose values are:
 * application
 
 
-## Cluster specification
+## Clusterspec
```

Comment:
I think cluster specification is intentionally vague. `ClusterSpecs` are gen1 construct and we will need to replace it with `base cluster` or `cluste r template`

Reply:
For now should I let it be ```Clusterspec``` or should we change it to `cluster template` now itself ?

Reply:
We are not switching to `cluster template` yet, and `cluster template` will replace `base cluster`, not `cluster spec`

Reply:
Ok